allure-report-face/src/helpers/splitUriList.js

```diff
@@ -0,0 +1,6 @@
+export default function splitUriList(uri) {
+    var os = require('os');
+    return uri.split(os.EOL).filter(function (el) {
+        return el.trim().length > 0 && !el.trim().startsWith('#');
+    });
+}
```

Comment:
1) This directory is for Handlebars helpers, not for utility functions.
2) In browsers, you don't need to care about line endings.
You imported whole os library because of constant: https://github.com/substack/os-browserify/blob/master/browser.js#L45